conf/distro/yogurt-vendor-connagtive-provisioning.conf

```diff
@@ -0,0 +1,15 @@
+require conf/distro/yogurt-vendor.conf
+
+DISTRO = "yogurt-vendor-connagtive-provisioning"
+DISTRO_NAME = "Yogurt Vendor Connagtive Provisioning (Phytec Vendor Distribution + Connagtive IoT Device Suite, used for initial provisioning)"
+
+# The distro feature "connagtive-provisioning" is only used for the initial
+# provisioning of a target system in the production stage.
+DISTRO_FEATURES += "connagtive-provisioning protectionshield"
```

Comment:
please remove distro feature protectionshield, because this activate the password setting

Reply:
Whoops, yes I will do that. That is actually a left-over of something I was trying out yesterday to remote the root password.